scripts/ci/images/specs.yaml

```diff
@@ -4,3 +4,5 @@ spack:
     checksum: false
   concretizer:
     unify: false
+  specs:
+    - c-blosc2
```

Comment:
@scottwittenburg I added here the dep

Reply:
Makes sense.  Hopefully just adding that in separately didn't change the cache hit rate when building the images?  Also hopefully it results in the adios2 configure finding it, but it seems it may need to be turned on instead of `Blosc`, eg [here](https://github.com/ornladios/ADIOS2/blob/afbe6d63feefa9085a3e2e6e7e84621c5124e338/scripts/ci/cmake/ci-ubuntu20.04-gcc8-serial.cmake#L12).

Reply:
Interestingly the ADIOS2 concretization override the `specs` packages (such as c-blosc2)